src/SourceBuild/content/eng/prep-source-build.sh

```diff
@@ -15,6 +16,21 @@
 ###   --runtime-source-feed       URL of a remote server or a local directory, from which SDKs and
 ###                               runtimes can be downloaded
 ###   --runtime-source-feed-key   Key for accessing the above server, if necessary
+###
+### Binary-Tooling options:
+###   --no-binary-tooling         Don't run the binary tooling
+###   --allowed-binaries          Path to the file containing the list of known binaries that are allowed
+###                               in the VMR and can be kept for source-building.
+###                               Default is src/installer/src/VirtualMonoRepo/allowed-binaries.txt
+###   --disallowed-sb-binaries    Path to the file containing the list of known binaries that are allowed
+###                               in the VMR but cannot be kept for source-building.
+###                               Default is null.
+###   --with-sdk                  Use the SDK in the specified directory
+###                               Default is the .NET SDK
+###   --packages-source-feed      URL or specified directory as the source feed for packages
```

Comment:
For consistency, this should probably be `--with-packages` to match the build script since it represents the same thing: https://github.com/dotnet/dotnet/blob/dc443600e100a3a1c379bed80737c3260ab07b7f/build.sh#L31